lib/svgo/css-select-adapter.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+/**
+ * @param {any} node
+ * @return {node is any}
+ */
```

Comment:
No other types requires?

Reply:
The other types seemed to be compatible. Of course there are some annotations missing, but because all of them are treated as `any` then it works, as it confirms to the adapter interface.

Here it was needed because the `isTag` needs to return a type assertion.